ghost/core/core/frontend/src/member-attribution/member-attribution.js

```diff
@@ -0,0 +1,90 @@
+// Location where we want to store the history in localStorage
+const STORAGE_KEY = 'ghost-history';
+
+// How long before an item should expire (24h)
+const TIMEOUT = 24 * 60 * 60 * 1000;
+
+// Maximum amount of urls in the history
+const LIMIT = 15;
```

Comment:
How did you come to this number?

Reply:
For now, this is arbitrary. I think 15 will be more than enough, or rather too much. We can later update this number after we've come up with our Last Post Algorithm™️. I think that will give us more information to determine this number. What do you think?

Reply:
Okay cool - sounds good to me, pretty easy to change in future!

My initial thoughts were that it might be too small, like you could easily visit 15 pages in under 24 hours 🤔

Thinking about it more though - the chances of none of them being a Post or a Page is small, and that's all we really care about atm